apps/playground/src/app/components/indicators/wait/wait.component.ts

```diff
@@ -1,12 +1,52 @@
-import { Component } from '@angular/core';
+import { ChangeDetectorRef, Component } from '@angular/core';
+import { SkyWaitService } from '@skyux/indicators';
 
 @Component({
   selector: 'app-wait',
   templateUrl: './wait.component.html',
-  styleUrls: ['./wait.component.scss'],
 })
 export class WaitComponent {
   public isWaiting: boolean;
   public isFullPageWaiting: boolean;
+  public isServiceFullPageWaiting: boolean;
   public isNonBlocking: boolean;
+
+  #changeDetector: ChangeDetectorRef;
+  #waitSvc: SkyWaitService;
+
+  constructor(changeDetector: ChangeDetectorRef, waitSvc: SkyWaitService) {
+    this.#changeDetector = changeDetector;
+    this.#waitSvc = waitSvc;
+  }
+
+  public toggleFullPageWait(): void {
+    this.isFullPageWaiting = !this.isFullPageWaiting;
+
+    if (this.isFullPageWaiting) {
+      setTimeout(() => {
```

Comment:
The playground would never disable the full page wait previously. I added this timeout to turn off the wait after 5 seconds and denoted that on the trigger button as well.

The second method was added to allow for triggering a full page wait via the service as this is the only time we dynamically create the element.